### PR TITLE
Add operations builders to AuctionHouseBuildersClient

### PR DESCRIPTION
**Pull request by antey13**

### CODE REVIEW

**Review by zaxozhu — APPROVED** [review]

**Review by lorisleiva — CHANGES_REQUESTED** [review]
Thanks! Can we please keep these methods ordered alphabetically?

This is because these methods are just registering transaction builders so it removes the need to think in what order we want them when we register them here.

File: packages/js/src/plugins/auctionHouseModule/AuctionHouseBuildersClient.ts [hunk 1]
```diff
@@ -41,6 +45,15 @@ export class AuctionHouseBuildersClient {
     return createBidBuilder(this.metaplex, input);
   }
 
+  /** {@inheritDoc cancelBidBuilder} */
+  cancelBid(input: CancelBidBuilderParams) {
+    return cancelBidBuilder(input);
+  }
+  /** {@inheritDoc cancelListingBuilder} */
```
Comment (zaxozhu) [inline packages/js/src/plugins/auctionHouseModule/AuctionHouseBuildersClient.ts:52]: New line here, please

Comment (antey13) [reply packages/js/src/plugins/auctionHouseModule/AuctionHouseBuildersClient.ts:53]: done

**Comment by lorisleiva** [comment]
Thanks!

**Pull request merged**